docs/release-notes/march-2023.md

```diff
@@ -0,0 +1,145 @@
+
+# Semgrep Code
+
+- **Semgrep Pro Engine beta** toggle is enabled by default in the [Semgrep Editor](https://semgrep.dev/orgs/-/editor/) and [Semgrep Playground](https://semgrep.dev/playground). Rules can still run with the Semgrep OSS Engine if `interfile: true` is not specified in the rule.
+- Findings from Pro rules or Semgrep Pro Engine are now labeled with a gem icon to let you know where the finding has come from.
+    ![semgrep-code-findings-pro-rule-gem.png](/img/release-notes-semgrep-code-findings-pro-rule-gem.png)
+    
+
+## Semgrep Pro Engine
+
+### Added
+
+- Previously, when installing Semgrep Pro Engine, Semgrep CLI downloaded the most recently released version of Semgrep Pro Engine. As a consequence, this version of Semgrep Pro Engine might not have been the most compatible version with Semgrep OSS Engine. With this update, the most compatible version of Semgrep Pro Engine with Semgrep OSS Engine is downloaded during the installation.
+    This behavior is only supported for Semgrep version 1.12.1 and later. Previous versions still download the most recently released version, as before.
+    
+- taint-mode: Semgrep Pro Engine’s taint analysis capabilities for Java now include support for basic field sensitivity through getters and setters. If you call `obj.setX(tainted)`, Semgrep can now identify that a subsequent call to `obj.getX()` will carry the same taint as `tainted`. Moreover, Semgrep can differentiate between different fields accessed by the getters and setters, such as `obj.getX()` and `obj.getY()`.
+    It's important to note that Semgrep Pro Engine doesn't examine the definitions of the getter and setter methods, and it doesn't know whether other methods like `obj.clearX()` clear the taint that `obj.setX(tainted)` adds. Nonetheless, this new feature enables Semgrep to detect vulnerabilities more accurately in tainted data flow in Java code.
+    
+
+### Changed
+
+- CI scans that use Semgrep Pro Engine now run intrafile and interprocedural taint analysis by default in differential scans (such as PR or MR scans). Note that interfile analysis is not run
+in differential scans for performance reasons.
+
+## Semgrep Cloud Platform
+
+### Added
+
+- For organizations with role-based access control (RBAC) enabled, members are now able to [log in through the CLI](https://semgrep.dev/docs/semgrep-app/user-management/#member-scoped-access-tokens) and send findings data from their local machine to the Semgrep Cloud Platform.
+
+## Semgrep Supply Chain
+
+### Added
+
+- You can now receive Semgrep Supply Chain notifications in your Slack channel. Be notified of **reachable vulnerabilities** as soon as a scan finishes. Install the [Semgrep Slack app](https://returntocorp.slack.com/apps/A03BM27NJ0H-semgrep?tab=more_info), then type `/semgrep_subscribe` to start setting up your Slack notifications.
```

Comment:
```suggestion
- You can now receive Semgrep Supply Chain notifications in your Slack channel. Be notified of **reachable vulnerabilities** as soon as a scan finishes. Sign in to Semgrep Cloud Platform and click Settings > Integrations > Add integration > Slack and follow the instructions to start setting up your Slack notifications.
```

I made this change to my own text because this is the preferred method to add integrations (better to do it through SCP than through Slack on the browser - the user might not be logged in to their Slack workspace in the browser, but is logged in their Slack App.)